pyproject.toml

```diff
@@ -52,6 +52,7 @@ doc = [
   "numpydoc",
   "openneuro-py",
   "pandas",
+  "pathlib",
```

Comment:
it's part of the standard library and doesn't need to be listed here

Reply:
thx, this comes up though :thinking: 

```
/home/circleci/project/mne_bids/path.py:docstring of mne_bids.BIDSPath.root:1: WARNING: py:class reference target not found: pathlib._local.Path [ref.class]
```

Reply:
See https://github.com/mne-tools/mne-bids/issues/1358#issuecomment-2557623010 . #1353 includes a work around for this but hasn't been merged yet